sdk/core/src/shared/SdkErrorFromCode.ts

```diff
@@ -1,45 +1,14 @@
-export default class SdkError extends Error {
-  private readonly _code: string
-  private readonly _context: any
-  private readonly _originalError: any
-  private readonly _httpStatusCode: any
+import { SdkError } from '@affinidi/common'
 
-  constructor(codeOrError: any, context: any = {}, originalError: any = {}) {
-    const isCode = typeof codeOrError === 'string'
-
-    const code = isCode ? codeOrError : codeOrError.code
-    const error = isCode ? SdkError.errors[code] : codeOrError
+export default class SdkErrorFromCode extends SdkError {
```

Comment:
What is the context behind this change? I'm a bit hesitant about naming and this kind of nesting in general, looks like OperationError.

Reply:
`internal-api-clients` throws `SdkError` in some cases, so `SdkError` had to be moved to `common`.

However, specific SDK error codes (`COR-XXX`) clearly do not belong in `common`, so I left this functionality in `wallet-core-sdk`.